src/core/token/ERC1155Core.sol

```diff
@@ -145,12 +144,6 @@ contract ERC1155Core is ERC1155, ModularCore, Multicallable {
             selector: BeforeApproveForAllCallback.beforeApproveForAll.selector,
             mode: CallbackMode.OPTIONAL
         });
-        supportedCallbackFunctions[5] =
```

Comment:
is this intended?

Reply:
mistake on my end, updated to add this back in